packages/service-utils/src/node/logging.ts

```diff
@@ -0,0 +1,36 @@
+import type { ServerResponse } from "http";
+import { AuthInput, extractAuthorizationData } from ".";
```

Comment:
should extract this from `index.ts` as well otherwise we'll end up with cyclic dependencies